components/uniset/src/builder.rs

```diff
@@ -16,14 +16,14 @@ pub struct UnicodeSetBuilder {
     intervals: Vec<u32>,
 }
 
-impl UnicodeSetBuilder {
+impl<'d> UnicodeSetBuilder {
     /// Returns empty [`UnicodeSetBuilder`]
     pub const fn new() -> Self {
         Self { intervals: vec![] }
     }
 
     /// Returns a [`UnicodeSet`] and consumes the [`UnicodeSetBuilder`]
-    pub fn build(self) -> UnicodeSet {
+    pub fn build(self) -> UnicodeSet<'d> {
```

Comment:
When building a UnicodeSet, you can set the lifetime to `'static` since it doesn't borrow anything